From f869dee3954a674b4a2cfc9811e24463e7f5312c Mon Sep 17 00:00:00 2001
From: Sofi <sofi+git@mailbox.org>
Date: Wed, 29 Mar 2023 10:45:03 +0200
Subject: [PATCH 2/2] Update to follow auto-generated standard

---
 data/isdv4-52d3.tablet | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/data/isdv4-52d3.tablet b/data/isdv4-52d3.tablet
index a9b97ec..2797113 100644
--- a/data/isdv4-52d3.tablet
+++ b/data/isdv4-52d3.tablet
@@ -8,14 +8,12 @@ Name=Wacom HID 52D3
 ModelName=WACF2200
 DeviceMatch=i2c:056a:52d3
 Class=ISDV4
-IntegratedIn=Display;System
 Width=12
 Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
 
 [Features]
-Reversible=false
 Stylus=true
 Touch=true
-Ring=false
 Buttons=0
-BuiltIn=true
-- 
2.39.2

